scripts/get_version.py

```diff
@@ -2,52 +2,72 @@
 from VSTSRequest import APIRequest
 
 def get_builds(response):
-    print(f'status code: {response.status_code}')
+    print('status code: {r.status_code}'.format(r=response))
```

Comment:
strange that's less readable to me. what was wrong wit the f string?

Reply:
f strings are only supported in python 3.6 and above. 3.6 isn't in Ubuntu 16.04's package repositories so we need to add a custom ppa key to get it in the setup script which can a long time and isn't officially supported. If we were running on 18.04 3.6 is installable by default but it is easiest to run python 3.5 which is already installed.